src/gamma_inc.jl

```diff
@@ -0,0 +1,573 @@
+using Base.Math: @horner
+using StatsFuns
```

Comment:
When adding libraries they need to also be added to the `REQUIRE` file.

However I assume is for `logmxp1`? In that case, it might make sense to move the function to here.

Reply:
Yes this is for `logmxp1`

Reply:
But we will require more of such functions from `StatsFuns` in future maybe. So won't it be good to add it at once in `REQUIRE`? (Like `StatsFuns` has `SpecialFunctions` added too.)